Assets/Plugins/LeapMotion/Modules/Hands/Scripts/HandBinder/HandBinderAutoBinder.cs

```diff
@@ -110,24 +104,40 @@ public static class HandBinderAutoRigger {
         /// <param name="bones">The bones you want to sort through</param>
         /// <param name="isThumb">Is it a thumb</param>
```

Comment:
Why does the caller need to indicate if it's a thumb?

Reply:
useMeta - could be ambiguous - e.g. is meta - metadata or meta - metacarpal?

Reply:
Leap data defines the thumb with a meta and proximal bone in the exact same place making it 4 bones for the thumb. This make it a special case for a rigged hand which usually only has 3 bones for the thumb not 4. So given the incoming rigged hand only has 3 fingers it will assign the proximal bone as the first found bone for the thumb rather than the meta.